R/display.R

```diff
@@ -1,5 +1,7 @@
 #' Display message on the top-right corner
 #'
+#' @note You can disable shiny.info by setting options(shiny.info = FALSE).
```

Comment:
How about adding this note to:
1) README
2) Console messages after installing / loading the package
?